about.html

```diff
@@ -0,0 +1,161 @@
+                        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem
+                            doloremque aut corporis molestias, voluptates voluptatibus itaque perferendis libero sapiente quis
+                            aspernatur quaerat dolor eaque autem nesciunt perspiciatis inventore. Cupiditate.</p>
+                        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    <img src="/images-capstone/web-summit-logo.png" alt="about me section logo"/>
+        </div>
+        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit:</p>
+        <p><a href="mailto:websummitlisbon@gmail.com" class="about-email">websummitlisbon@gmail.com</a></p>
+    </section>
+    <!--end of about info section-->
+
+        <!--past events section-->
+<section id="past-events">
+    <div id="past-events-container">
+        <h2 class="see-past-events">See past CC Global Summits events</h2>
+        <hr>
+        <p>Check previous events and summits</p>
+        <ul id="events">
+            <li>
+                <!-- <div class="overlay-img"> -->
+                <img class="previous-web-summit" src="/images-capstone/web-summit-2021.png" alt="web summit 2021">
+                    <div class="overlay">
+                        <h3>2021</h3>
+                        <p>Web Summit Rio de Janeiro</p>
+                    </div>
+            <!-- </div> -->
+            </li>
+
+            <li>
+                <!-- <div class="overlay-img"> -->
+                <img class="previous-web-summit" src="/images-capstone/web-summit-2020.png" alt="web summit 2020">
+                    <div class="overlay">
+                        <h3>2020</h3>
+                        <p>Web Summit Toronto</p>
+                    </div>
+            <!-- </div> -->
+            </li>
+        </ul>
+    </div>
+</section>
+        <!--end of past events section-->
+    <footer id="second-copyright">
+        <div id="second-footer-container">
+            <div class="second-footer-logo">
+                <img src="images-capstone/creative-commons-logo.png" alt="creative commons" id="cc-logo-black"/>
+                <img src="images-capstone/cc-logo-white.png" alt="Creative Commons" id="cc-logo-white">
+            </div>
+            <div class="second-cc-info">
+                <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img
+                            alt="Creative Commons License"
```

Comment:
- [ ] In the mobile view of `about.html` you have missed including the `sponsor` section. Please include this.

### Missed sponsor section in about.html(mobile view)
![Screenshot from 2022-06-24 01-58-18](https://user-images.githubusercontent.com/26767383/175394000-3382cdd0-7a26-483b-984f-6b9bb8accb28.png)